tools/clang/unittests/HLSLExec/ShaderOpTest.cpp

```diff
@@ -861,6 +861,11 @@ void ShaderOpTest::CreateShaders() {
       CHECK_HR(pLibrary->CreateBlobWithEncodingFromPinned(
           pText, (UINT32)strlen(pText), CP_UTF8, &pTextBlob));
       CHECK_HR(m_pDxcSupport->CreateInstance(CLSID_DxcCompiler, &pCompiler));
+      WEX::Logging::Log::Comment(L"Compiling shader:");
```

Comment:
Added this for convenience. Was handy when validating.

Reply:
Thanks this is great